### DISCUSSION TITLE
Not sure what should happen here (Hypothetical)

### DISCUSSION THREAD

**Discussion by computamike** [openbenches/openbenches.org#346] [Ideas]
What happens if I go to a bench - and it isn't there?  I know that there are some benches in my local - well - I'll say park ([Himley Hall](https://www.dudley.gov.uk/see-and-do/himley-hall/)) - and these benches aren't necessarily bolted to the floor -and might be moved if the area was being used for an event (and the bench may not be put back in the same location after)..  Is this something that could happen?  What should I do as a user when I encounter (or in this case don't encounter) a bench that should be there - but isn't.

**Comment by edent** [comment]
This is a problem we anticipated with the database design. There is a field called `present` which can be used to indicate the bench no longer exists.

However, we don't have a user interface for setting that...

So, the options are:

* Do nothing
* Add a comment to the page saying the bench has gone
* Edit the bench and move it to a different location (if it still exists)
* Edit the bench and use the delete option
* Edit the bench and use the (not yet implemented) "this bench used to be here but it has disappeared" option

Which do you think is best?

**Comment by computamike** [comment]
I think that the ability to edit an mark it as "not present" would be good.  If it's a case that the bench has been swapped or moved- and I could reposition it - then that's good too - because the edit function lets me do that.

Not being overly familiar with the code - I've just been checking - and "Not Present" benches are still potentially searchable through the search inscriptions functionality - and should be displayable - so that's good!.

If someone marks a bench that someone else has uploaded, then that would reduce that uploader's score for the leader board?
